pkg/util/netutil/netutil.go

```diff
@@ -14,14 +14,13 @@
 
 package netutil
 
-import (
-	"github.com/phayes/freeport"
-)
+import "net"
 
 func GetFreePort() (int, error) {
-	port, err := freeport.GetFreePort()
+	l, err := net.Listen("tcp", ":0")
```

Comment:
Listen `:0` could make this port open to the public.

```suggestion
	l, err := net.Listen("tcp", "127.0.0.1:0")
```

Reply:
BTW, why not use `net.Listen("tcp", ":0")` directly? We can pass a `Listener` to ssh client which is more common in golang.

@gaocegege

Reply:
> Listen `:0` could make this port open to the public.

At the end of this function, the listener will be closed directly, ":0" or "127.0.0.1:0" are the same.

Reply:
> BTW, why not pass a `Listener` to the ssh client directly? Which is more common in golang?
> 
> @gaocegege

Actually, the ssh client code is copied from https://github.com/okteto/okteto.

I think passing Listener also works but okteto's package passes the port.